contracts/src/interfaces/IGateway.sol

```diff
@@ -69,8 +69,8 @@ interface IGateway {
 
     /// @dev Emitted once the funds are locked and an outbound message is successfully queued.
     event TokenSent(
-        address indexed token,
```

Comment:
Nice catch! Though can we rather keep `token` as the first parameter (since the event is named `TokenSent`, so `token` is most important), and instead update the code in `sendToken` that emits the event?

Reply:
Addressed in c8e71a3df8a1d0ea584437e45b3955d0fb3507d3